dedibox-console/account/how-to/resolve-abuse.mdx

```diff
@@ -1,32 +1,64 @@
 ---
 meta:
-  title: How to resolve abuse tickets
+  title: How to resolve Dedibox abuses and cases
   description: This page explains how to resolve Abuse tickets.
 content:
-  h1: How to resolve abuse tickets
+  h1: How to resolve Dedibox abuses and cases
   paragraph: This page explains how to resolve Abuse tickets.
 tags: resolve abuse complaint ticket dedibox-console 
 dates:
-  validation: 2024-02-01
+  validation: 2024-02-19
   posted: 2020-03-12
 categories:
   - account
 ---
 
-Abuse tickets are complaints regarding the hosting of copyrighted content or engaging in illegal activities using Scaleway services.
+Abuse tickets are complaints regarding the use of Scaleway Dedibox services for illegal behavior or hosting of illegal content. Abuses for the same issue on the same resource within a given timeframe are grouped by case.
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the lock of their services.
```

Comment:
```suggestion
Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the locking of their services.
```